recipes/cose-c/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "cci.20200430":
```

Comment:
I see some commits by @gocarlos since 20200430 at https://github.com/cose-wg/COSE-C/commits/master
Do you have an idea why current master does not work with our packaged version of cn-cbor?

Reply:
API breaks 💣

Reply:
We converted cose c to cpp, so we need now a cpp compiler

Reply:
Thanks for the reply!